langchain/llms/databricks_endpoint.py

```diff
@@ -0,0 +1,224 @@
+from abc import ABC, abstractmethod
+import logging
+import os
+from typing import Any, Dict, List, Mapping, Optional, Union, Callable
+
+from pydantic import BaseModel, Extra, Field, validator, root_validator, ValidationError, PrivateAttr
+import requests
+
+from langchain.llms.base import LLM
+from langchain.llms.utils import enforce_stop_tokens
+
+__all__ = ["DatabricksEndpoint"]
+
+
+class DatabricksClientBase(BaseModel, ABC):
+    api_url: str
+    api_token: str
+
+    def post_raw(self, request):
+        headers = {"Authorization": f"Bearer {self.api_token}"}
+        response = requests.post(self.api_url, headers=headers, json=request)
+        # TODO: error handling and automatic retries
+        assert response.ok, f"HTTP {response.status_code} error: {response.text}"
+        return response.json()
+    
+    @abstractmethod
+    def post(self, request):
+        raise NotImplementedError
+
+
+class DatabricksServingEndpointClient(DatabricksClientBase):
```

Comment:
Nit: simple doc comment for `DatabricksServingEndpointClient` & `DatabricksClusterDriverProxyClient`